google/cloud/bigtable/ci/run_conformance_tests_proxy_bazel.sh

```diff
@@ -35,14 +35,19 @@ bazel_args=("$@")
 # Building the binary can easily take more than the timeout limit.
 pushd "$(dirname "$0")/../test_proxy" >/dev/null
 "${BAZEL_BIN}" build "${bazel_args[@]}" :cbt_test_proxy_main
-nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 &
+nohup "${BAZEL_BIN}" run "${bazel_args[@]}" :cbt_test_proxy_main -- 9999 > proxy.log &
```

Comment:
I think we want the logs to go the same file. That way we can correlate the C++ library side logging with the test that is being run.

I also think we would want to write to a temporary file.

I think you should revert the logging changes in this file. We can deal with the logs in a follow up PR.

Reply:
Done.